src/views/CreateBlog.vue

```diff
@@ -1,9 +1,48 @@
 <template>
+        <div class="flex justify-center mt-10" v-if="successMsg">
+            {{ successMsg }}
+        </div>
     <div class="flex justify-center font-bold text-3xl mt-20">
-        Where user will create new blog
+        <FormKit
+            type="form"
+            submit-label="Create Blog"
+            @submit="CreateBlog"
```

Comment:
should be renamed to `createBlog` since this is a widely accepted convention to name Constructor Functions   (https://www.w3schools.com/js/js_object_constructors.asp) with Capital First Letter.